pylint/checkers/base/name_checker/checker.py

```diff
@@ -17,6 +17,7 @@
 from typing import TYPE_CHECKING
 
 import astroid
+import astroid.bases
```

Comment:
`astroid.bases` is referenced in this file. It doesn't fail currently likely because other files also import it. Just add it here as well for good measure.